lib/converters.js

```diff
@@ -25,6 +25,22 @@ function indent(content, spaces, ignoreFirst) {
     }).join("\n");
 }
 
+function emphasisFormats( text ) {
+  return {
+    strong: '**',
+    italic: '*',
+  };
+}
+
+function parseTextFormat( text ) {
+  var formats = emphasisFormats();
```

Comment:
Why not doing `formats = { ... }` here instead of calling this function? :thought_balloon: